src/ui/theme/theme.ts

```diff
@@ -102,8 +74,7 @@ export const theme = extendTheme({
     heading: openSauce.style.fontFamily,
   },
   sizes: {
-    // 4.5: '1.125rem',
```

Comment:
These should be exactly the same

Reply:
not sure I fully understood

Reply:
Divide 1.125 by 1.125 that becomes 1 rem. Divide 4.5 by 1.125 and that becomes 4

Reply:
Perhaps, this was working before because 4.5 was getting exactly translated into 1.125 but if the goal is to have a scale for mapping pure numbers to rem then I think some of our components that set the size and also height and width and minwidth and minheight might be messing up. I ran into this when fixing our status bubbles in my ui updates PR